cpp/oneapi/dal/backend/primitives/stat/cov_dpc.cpp

```diff
@@ -51,6 +51,98 @@ sycl::event means(sycl::queue& q,
     });
 }
 
+// Subtract 1d array from 2d array elementwise
+template <typename Float>
+sycl::event elementwise_difference(sycl::queue& q,
```

Comment:
Please use elementwise operations that are [already present](https://github.com/oneapi-src/oneDAL/blob/master/cpp/oneapi/dal/backend/primitives/element_wise.hpp)